### ISSUE TITLE
Key error when importing computers

### ISSUE THREAD

**Issue by szoupanos** [aiidateam/aiida-core#1806]
The following  error may appear when importing a computer.
More information will be added soon.

```
import_nodes_uuid = set(v['uuid'] for v in data['export_data'][NODE_ENTITY_NAME].values())
KeyError: 'Node'
```

**Comment by ltalirz** [comment]
@szoupanos Please don't forget to "add more information".

Are you going to work in this in the next days?
If not I will remove it from the 0.12.3 milestone

**Comment by ltalirz** [comment]
Ok, this is not officially released but was fixed by @CasperWA in #3315. Closing...
